recipes/libpcap/all/conanfile.py

```diff
@@ -29,11 +29,13 @@ class LibPcapConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "enable_libusb": [True, False],
+        "with_snf": [None, True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "enable_libusb": False,
+        "with_snf": None,
```

Comment:
```suggestion
        "with_snf": True,
```

Reply:
After reading through the configure file, I think this should default to `False` (though maybe `None` still makes sense) since I believe the "yes, if present" means that `--with-snf=yes` and `--with-snf` are equivalent.

Reply:
Hi, I was reading through the libpcap stuff more and I do believe None is a valid option here, when `--with-snf` is not present, the autoconfigure stuff will attempt to see if the snf library is present at `/opt/snf` and if so link against it. This is different behavior than explicitly passing yes/no

Reply:
I am not sure how to best model this with conan2, if it is even possible? Maybe defaulting to false/true is sufficient for this use case?